modules/ossm-member-roll-create.adoc

```diff
@@ -102,3 +101,51 @@ spec:
 ----
 +
 . Modify the default YAML to add your projects as `members`.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+[id="ossm-member-roll-create-member_{context}"]
+== Creating the {ProductName} members
+
+`ServiceMeshMember` resources are created by service mesh users who don't have privileges to add members to the Service Mesh Member Roll directly. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they cannot point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can grant users permissions to access the mesh by granting them the mesh-user user role, for example: 
+
+----
+$ oc policy add-role-to-user -n <control-plane-namespace> --role-namespace <control-plane-namespace> mesh-user <user-name>.
+----
+
+Administrators can modify the mesh users role binding in the control plane project to specify the users and groups that are granted access. The `ServiceMeshMember` adds the project to the `ServiceMeshMemberRoll` within the control plane project it references. 
+
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+spec:
+  controlPlaneRef:
+    namespace: control-plane-namespace
+    name: minimal-install
+----
+
+The mesh users role binding is created after the administrator creates the `ServiceMeshControlPlane` resource. An adminstrator can use the following command to add a role to a user.
```

Comment:
"is created" is passive voice, it's unclear if this happens automatically or requires user intervention.  

When a cluster administrator creates the  `ServiceMeshControlPlane` the Operator automatically creates the mesh users role binding. (I assume we're talking about a cluster-admin role?)  The cluster administrator can then use the following command to add a role to a user.